src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -52,38 +43,36 @@ const Inventory = ({
     sort: '-last_seen',
     name: '',
   });
+  const [fullFilters, setFullFilters] = useState();
   const [total, setTotal] = useState(0);
   const entities = useSelector(({ entities }) => entities || {});
+
   const addNotification = (data) => dispatch(notification(data));
   const [disableRuleModalOpen, setDisableRuleModalOpen] = useState(false);
   const [curPageIds, setCurPageIds] = useState([]);
   const [pathwayRulesList, setPathwayRulesList] = useState();
   const [pathwayReportList, setPathwayReportList] = useState();
+  const [loading, setLoading] = useState();
```

Comment:
Here as well, I would suggest renaming 'loading' to isLoading.

Reply:
I agree, thats a better name for it